client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -12,102 +12,135 @@ import wrapWithClickOutside from 'react-click-outside';
  */
 import { transferStates } from 'state/automated-transfer/constants';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import {
+	getAutomatedTransferStatus,
+	isAutomatedTransferTransferring,
+} from 'state/automated-transfer/selectors';
 import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
 
 class PluginAutomatedTransfer extends Component {
 
 	static propTypes = {
+		isTransferring: PropTypes.bool,
 		plugin: PropTypes.object,
-		status: PropTypes.string,
+		transferState: PropTypes.string,
 		translate: PropTypes.func,
-	}
+	};
 
 	static defaultProps = {
 		plugin: {
 			name: 'Plugin',
 		},
-	}
+	};
 
 	state = {
 		clickOutside: false,
+		shouldDisplay: true,
+		transferComplete: false,
+	};
+
+	componentWillMount() {
+		if ( transferStates.COMPLETE === this.props.transferState ) {
+			this.setState( { shouldDisplay: false } );
+		}
 	}
```

Comment:
Wait @dmsnell!
The version you're looking is plain wrong!

I've found out several different cases were the component doesn't show or hide as intended, and I'm working on it right now.